### PR TITLE
[example] Add face recognition with facenet model (part web)

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: cuiyan <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@belem @BruceDai please review

File: examples/face_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# WebNN API Face Recognition Example
+This example loads Face Detection models (trained by WIDERFace) and Face Recognition model (trained by davidsandberg) in TensorFlow format, constructs and inferences them by WebNN API.
```
[the remark this answers is not in the document]
Comment (cuiyanx) [reply examples/face_recognition/README.md]: The format of original FaceNet model is tensorflow(.pb) that trained by davidsandberg, we just covert this model to OpenVino format.

File: examples/face_recognition/model/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Get models
+Download [ssd_mobilenetv1_face](https://drive.google.com/file/d/14hxWR1AFEQRfnExSm42R4ntYGX4QKMZz/view?usp=sharing), [ssd_mobilenetv2_face](https://drive.google.com/file/d/1mDDCJVZyaIZXz6bpJE_zF1C3nZCH45qD/view?usp=sharing),  [ssdlite_mobilenetv2_face](https://drive.google.com/file/d/1QAFPChUVU4MgQwQSO7n1ac0BGpQAmmBK/view?usp=sharing), [tiny_yolov2_face](https://drive.google.com/file/d/1PWW2LxKXPSlW-X4epEFITqJOYrgo3lep/view?usp=sharing). These models are used for face detection, please put them to facial_landmark_detection/model. And Download [facenet](https://github.com/cuiyanx/webml-polyfill/tree/add-face-recognition-example-with-facenet-openvino/examples/face_recognition/model). This model is used for face recognition, please download it here.
```
[the remark this answers is not in the document]
Comment (cuiyanx) [reply examples/face_recognition/model/README.md]: OK, I will do this.

**Comment by cuiyanx** [comment]
@ibelem please review again.

**Comment by ibelem** [comment]
Close this PR as @cuiyanx will refactor the code.

**Pull request closed without merging**